### PR TITLE
(#4071) - Ignore ie polyfill

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

File: lib/deps/functionName.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ if ((function f() {}).name) {
     return fun.name;
   };
 } else {
+  // We dont run coverage in IE
+  /* istanbul ignore next */
```
Comment (nolanlawson) [inline lib/deps/functionName.js]: I believe you can do `istanbul ignore else` to ignore the whole else block

**Comment by daleharvey** [comment]
Updated to use `ignore else`, also skipped ci since there is no point in running the full suite for a comment change, will verify it does skip in coverage though

**Comment by nolanlawson** [comment]
:+1:

**Comment by daleharvey** [comment]
Sweet cheers - https://github.com/pouchdb/pouchdb/commit/137376d5cb59ff29bafe522b7263fe4e659192e5

**Pull request closed without merging**